etc/telegraf/telegraf.toml.j2

```diff
@@ -29,6 +29,7 @@
     commands = ["echo {{ is_enabled }}"]
     name_override = "integration_usage"
     interval = "6h"
+    round_interval = false
```

Comment:
A note on why we overriding this would be helpful

Reply:
removed this already